### PR TITLE
Ejercicio 4 stats sentences

### LINKED ISSUE #4: statsSentences.sh
Indicador estadístico de longitud de oraciones (la más corta, la más larga y el promedio de longitud).

### PR DESCRIPTION
**Pull request by salvanya**

### CODE REVIEW

File: statsSentences.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+### corresponding with the least and the most amount of words in a sentence. Also we add 1 to the sentence
+### counter and the amount of words to the word total, before restarting the word counter.  
+### Finally we calculate the average and then the results are printed.
+
+FILE=$1
+
+FIRST_ITERATION=true
+WORD_COUNTER=0
+
+while read -ra LINE;
```
Comment (fedeantuna) [inline statsSentences.sh]:
```suggestion
while read -ra LINE
```

File: statsSentences.sh [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            fi            
+
+            let SENTENCE_COUNTER++
+            WORD_ACCUMULATOR=$( expr $WORD_ACCUMULATOR + $WORD_COUNTER)
+            WORD_COUNTER=0
+            continue        
+        fi        
+
+        let WORD_COUNTER++
+    done;
```
Comment (fedeantuna) [inline statsSentences.sh]:
```suggestion
    done
```

File: statsSentences.sh [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+### Finally we calculate the average and then the results are printed.
+
+FILE=$1
+
+FIRST_ITERATION=true
+WORD_COUNTER=0
+
+while read -ra LINE; 
+do
+    for WORD in "${LINE[@]}";
```
Comment (fedeantuna) [inline statsSentences.sh]:
```suggestion
    for WORD in "${LINE[@]}"
```

File: statsSentences.sh [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+            continue        
+        fi        
+
+        let WORD_COUNTER++
+    done;
+done < $FILE
+
+WORD_AVERAGE=$( expr $WORD_ACCUMULATOR / $SENTENCE_COUNTER )
+
+#echo "The number of lines is $SENTENCE_COUNTER"
```
Comment (Flor243) [inline statsSentences.sh]:
```suggestion

```
When working with git, it isn't necessary make versioning with comments

**Review by fedeantuna — APPROVED** [review]

**Review by Flor243 — APPROVED** [review]

**Pull request merged**